spec/vector_spec.rb

```diff
@@ -921,6 +921,18 @@
         end
       end
 
+      context "#to_df" do
+        let(:dv) { Daru::Vector.new(['a','b','c'], name: :my_dv, index: ['alpha', 'beta', 'gamma']) }
+        let(:df) { dv.to_df }
+        it 'converts the vector to a single-vector dataframe' do
+          expect(df[:my_dv]).to eq dv
+        end
```

Comment:
Two considerations:
* in fact, you never check df is really DataFrame instance, not something other
* what about resulting dataframe name?..